jobs/load-hiv-data-from-openmrs.yml

```diff
@@ -28,3 +28,6 @@ configuration:
           - tableName: "pmtct_pregnancy"
           - tableName: "tb_lab_results"
           - tableName: "tb_screening"
+          - tableName: "users"
+          - tableName: "user_roles"
+          - tableName: "user_logins"

```

Comment:
why add these new tables to both common and hiv data jobs?  In all Haiti environments these both run, so no need for them to run twice?  Am I missing something?